app/react/portainer/environments/wizard/EnvironmentsCreationView/WizardKubernetes/KubeConfigTeaserForm.tsx

```diff
@@ -0,0 +1,100 @@
+import { Field, Form, Formik } from 'formik';
```

Comment:
our tendency in these would be to use the same component as in EE, but add a check ` const limitedToBE = isLimitedToBE(option.feature)` which will decide if form is readonly/disable. the purpose in the end is to have the same codebase for frontend

Reply:
Would you like me to use the exact file from EE in this way and add any missing types?

Reply:
potentially yes. and the same for EE. should use the exact file as in CE